libs/platform/src/lib/components/footer/footer.component.scss

```diff
@@ -0,0 +1,208 @@
+$fd-copy-right-font-color: var(--sapField_PlaceholderTextColor);
+$fd-link-font-color: var(--sapLinkColor);
+$fd-footer-font-size: var(--sapFontSize, 0.875rem);
+
+@mixin fd-reset {
+    font-size: var(--sapFontSize);
+    line-height: var(--sapContent_LineHeight);
+    color: var(--sapTextColor);
+    font-family: var(--sapFontFamily);
+    font-weight: normal;
+    box-sizing: border-box;
+    border: 0;
+
+    &::before,
+    &::after {
+        box-sizing: inherit;
+        font-size: inherit;
+    }
+}
+@mixin fd-rtl {
+    @at-root {
+        [dir='rtl'] &,
+        &[dir='rtl'] {
+            @content;
+        }
+    }
+}
+
+.fdp-footer {
+    @include fd-reset();
+
+    width: 100%;
+    background-color: var(--sapField_Background);
+    display: inline-block;
+    padding: 1.5rem 3rem 2rem 3rem;
+
+    &--logo {
+        width: 3rem;
+        height: 1.5rem;
+        overflow: hidden;
+    }
+    &--link {
+        color: $fd-link-font-color;
+        font-family: var(--sapFontFamily, '72', '72full', Arial, Helvetica, sans-serif);
+        font-size: 0.875rem;
+        display: inherit;
+        padding-top: 0.75rem;
+        width: 73%;
+        > div > * {
+            margin-right: 1.5rem;
```

Comment:
i am still not sure if applying `> div > *` is a safe way to do this. Can it not be more specific? If we know a link will contain `fdp-link` only, can we not try to target `fdp-link` directly?
also, please move 1.5rem to common variable and other redundant values to common variables

Reply:
this is not resolved ⬆️